### PR TITLE
{Column,Index}ID is different from ID

### PR DESCRIPTION
**Pull request by tamird**
Didn't quite do this right the first time.

### CODE REVIEW

File: structured/structured.go [hunk 1]
```diff
@@ -28,6 +28,12 @@ import (
 // ID is a custom type for {Database,Table}Descriptor IDs.
 type ID uint32
 
+// ColumnID is a custom type for DatabaseDescriptor IDs.
```
Comment (petermattis) [inline structured/structured.go]: `s/DatabaseDescriptor/ColumnDescriptor/g`

File: structured/structured.go [hunk 2]
```diff
@@ -28,6 +28,12 @@ import (
...
+type ColumnID uint32
+
+// IndexID is a custom type for DatabaseDescriptor IDs.
```
Comment (petermattis) [inline structured/structured.go]: `s/DatabaseDescriptor/IndexDescriptor/g`

**Comment by petermattis** [comment]
LGTM

I'm guessing you want to distinguish the IDs so that you can't accidentally use a `ColumnID` where an `IndexID` is expected. Seems reasonable, though you might want to call this out in the comments where the `ID` types are defined.

**Comment by vivekmenezes** [comment]
LGTM

**Comment by tamird** [comment]
> I'm guessing you want to distinguish the IDs so that you can't accidentally use a ColumnID where an IndexID is expected. Seems reasonable, though you might want to call this out in the comments where the ID types are defined.

Yep. Commented!

**Pull request merged**